Microservices/event-service/src/main/java/app/sportahub/eventservice/dto/response/EventResponse.java

```diff
@@ -1,15 +1,17 @@
 package app.sportahub.eventservice.dto.response;
 
+import app.sportahub.eventservice.enums.SkillLevelEnum;
 import app.sportahub.eventservice.model.event.Location;
 import app.sportahub.eventservice.model.event.Participant;
 import app.sportahub.eventservice.model.event.Team;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import java.time.LocalDate;
+import java.util.EnumSet;
 import java.util.List;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public record EventResponse(String id, String eventName, String eventType, String sportType, Location location,
```

Comment:
Can you replace the Location object in response with LocationResponse (you will need to create it).